src/components/Dashboard/FilingHistoryList/filings/ConsentContinuationOut.vue

```diff
@@ -26,7 +27,11 @@ export default class ConsentContinuationOut extends Vue {
 
   get expiry (): string {
     // FUTURE: expiry date may come from business object
-    return this.filing.data?.consentContinuationOut?.expiry || '[unknown]'
+    const expiry = this.filing.data?.consentContinuationOut?.expiry
+    if (expiry) {
+      return DateUtilities.apiToPacificDate(expiry)
```

Comment:
According to uxpin, this should be the long version ("August" not "Aug").

https://preview.uxpin.com/f3e9c0da57601f193d2f0c6b1946134b2ddeb2fe#/pages/160694550/comment/sitemap

Reply:
Beauty.